src/organisms/Breadcrumbs.test.jsx

```diff
@@ -0,0 +1,328 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { useContext, useEffect, useState } from "react";
+import { MemoryRouter } from "react-router-dom";
+import { GlobalStateProvider } from "../GlobalState";
+import { MAIN_ROUTES } from "../AppRoutes";
+import App from "../App";
+import Breadcrumbs from "./Breadcrumbs";
+
+test("renders breadcrumbs no url", async () => {
+  render(
+    <MemoryRouter initialEntries={[""]}>
+      <App />
+    </MemoryRouter>
+  );
+  await waitFor(() => {
+    const breadcrumbs = screen.queryByTestId("breadcrumbs-bar");
```

Comment:
Nice, these are great ways to incorporate waiting in tests so that we're not using hardcoded wait values which can produce unreliable test results.